doc/references.bib

```diff
@@ -100,3 +99,36 @@ @article{NolteEtAl2004
   volume = {115},
   year = {2004}
 }
+@article{Hosking1980,
+  author = {J. R. M. Hosking},
+  issn = {01621459},
+  journal = {Journal of the American Statistical Association},
+  number = {371},
+  pages = {602--608},
+  publisher = {[American Statistical Association, Taylor & Francis, Ltd.]},
+  title = {The Multivariate Portmanteau Statistic},
+  url = {http://www.jstor.org/stable/2287656},
+  volume = {75},
+  year = {1980}
+}
+
+@article{kilian_2006,
+  author = {Kilian, Lutz},
+  doi = {10.1017/S0266466606000442},
+  journal = {Econometric Theory},
+  number = {5},
+  pages = {961–967},
+  publisher = {Cambridge University Press},
+  title = {NEW INTRODUCTION TO MULTIPLE TIME SERIES ANALYSIS, by Helmut Lütkepohl, Springer, 2005},
+  volume = {22},
+  year = {2006},
+}
```

Comment:
I doubt this citation is correct. Maybe:

https://www.bibsonomy.org/bibtex/2560598c4abb7aa564be0f9fb161af2f4/fbw_hannover

Reply:
Added it from that link now.